docs/UserGuide.md

```diff
@@ -36,6 +37,20 @@ the user guide from start to end to fully understand how to use Class-ify. Howev
 * Refer to our [Command Summary](#6-command-summary) to have a quick overview of the different commands and their respective formats.
 * Refer to our [Glossary](#7-glossary) to learn key terms that are used in this User Guide.
 
+### 2.2 Navigating the Display
+This section introduces the different components of the display and their functions. 
+
+![Application Window](images/ClassifyApplicationWindow.png)
+
+| No. | Component             | Description                                                                                                                    |
+|-----|-----------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| 1   | Menu bar              | Provides convenient access to frequently used commands.                                                                        |
```

Comment:
```suggestion
| 1   | Toolbar              | Provides convenient access to frequently used commands.                                                                        |
```
To be consistent with the name used in the screenshot.

Reply:
Thanks Daniel. Didn't catch that 😅